src/utilities/apiCalls.js

```diff
@@ -0,0 +1,26 @@
+export const getUsers = async () => {
+    try {
+        const response = await fetch('https://turing-roomies-be.herokuapp.com/api/v1/users')
+        const checkedResponse = await checkResponse(response)
+        return checkedResponse
+    } catch (err) {
+        throw Error(err.message)
+  }
+}
+
+const checkResponse = response => {
+  if (response.ok) {
+    return response.json()
+  }
+  handleStatusError(response.status)
+}
+
+const handleStatusError = status => {
+  if (status === 404) {
+    throw Error('Sorry, page not found!')
+  }
+  if (status === 500) {
+    throw Error('Sorry, this page isn\'t working!')
+  }
+  throw Error('Sorry, something went wrong!')
+}
```

Comment:
I'm not sure what my opinion is on the await vs .then yet. I'm glad we did it this way, I wanted to learn the syntax. As of now I still prefer the .then syntax, but this works perfectly and the error handling is great!